parts/windows/containerdtemplate.toml

```diff
@@ -38,8 +38,8 @@ state = "C:\\ProgramData\\containerd\\state"
       [plugins.cri.containerd.default_runtime]
         runtime_type = "io.containerd.runhcs.v1"
         [plugins.cri.containerd.default_runtime.options]
-          Debug = true
-          DebugType = 2
+          Debug = false
```

Comment:
Do you have the test result how long we can retain the logs for containerd with these settings?